task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -47,6 +51,37 @@ spec:
         set -o pipefail
         source /utils.sh
 
+        declare -a ALLOWED_BASE_IMAGES=("registry.redhat.io/openshift4/ose-operator-registry")
```

Comment:
JFTR, I still don't like fact that those images should be hardcoded here

Reply:
I was suggesting to have it as parameter, this can be default value. So if users really want to, they can update it.

Reply:
but on the other hand, malicous image could be an attack vector, because we are going to run opm from the image in this task

Reply:
Right -- that attack vector is why the proposal was made. It is unfortunate that this required a Red Hat authorization to be able to run, but if we hear feedback for wanting to build without the authentication then we can consider broadening the restriction.